### DISCUSSION TITLE
TCP-based serial monitor?

### DISCUSSION THREAD

**Discussion by Xeevis** [fredlcore/BSB-LAN#585] [Q&A] [answered]
When using VS Code with Arduino extension there is selection between "Serial" and "TCP" serial monitors. Is it possible to enable TCP based serial monitoring on Olimex-EVB? Would probably ease some of my pains of hooking up the board via USB.

![image](https://github.com/fredlcore/BSB-LAN/assets/5835044/09b84e68-18e5-4bdb-b269-e1f3d72cace0)

**Comment by fredlcore** [comment, marked as answer]
Sounds like a great idea, but I have no idea how to do it. So if you can/want to do it, feel free to go ahead. Otherwise we already have monitor output via telnet. It only starts after the network is up and running, but after that, it's the same as over serial...

**Reply by Xeevis** [reply]
Yup after changing Debugging | Usage to Telnet I can connect serial monitor on port 23. That's all I needed, thanks!
